src/main/java/ui/components/KeyboardShortcuts.java

```diff
@@ -130,10 +130,17 @@
     public static final KeyCodeCombination CLOSE_PANEL =
             new KeyCodeCombination(KeyCode.W, KeyCombination.CONTROL_DOWN);
     
-    private static Set<KeyCodeCombination> populateJumpToFirstIssueSet() {
-        Set <KeyCodeCombination> result = new HashSet<KeyCodeCombination>();
-        result.add(new KeyCodeCombination(KeyCode.DIGIT1));
-        result.add(new KeyCodeCombination(KeyCode.DOWN, KeyCombination.CONTROL_DOWN));
+    private static Map<Integer, KeyCodeCombination> populateJumpTonthIssueMap() {
+        Map <Integer, KeyCodeCombination> result = new HashMap<>();
+        result.put(1, new KeyCodeCombination(KeyCode.DIGIT1, KeyCombination.CONTROL_DOWN));
+        result.put(2, new KeyCodeCombination(KeyCode.DIGIT2, KeyCombination.CONTROL_DOWN));
+        result.put(3, new KeyCodeCombination(KeyCode.DIGIT3, KeyCombination.CONTROL_DOWN));
+        result.put(4, new KeyCodeCombination(KeyCode.DIGIT4, KeyCombination.CONTROL_DOWN));
+        result.put(5, new KeyCodeCombination(KeyCode.DIGIT5, KeyCombination.CONTROL_DOWN));
+        result.put(6, new KeyCodeCombination(KeyCode.DIGIT6, KeyCombination.CONTROL_DOWN));
+        result.put(7, new KeyCodeCombination(KeyCode.DIGIT7, KeyCombination.CONTROL_DOWN));
+        result.put(8, new KeyCodeCombination(KeyCode.DIGIT8, KeyCombination.CONTROL_DOWN));
+        result.put(9, new KeyCodeCombination(KeyCode.DIGIT9, KeyCombination.CONTROL_DOWN));
```

Comment:
You can return an immutable view of the map using `Collections.unmodifiableMap`. Given the name, it's expected to be constant/immutable.